installing/installing_aws/installing-aws-account.adoc

```diff
@@ -21,6 +21,7 @@ include::modules/installation-aws-iam-user.adoc[leveloffset=+1]
 .Additional resources
 
 * See xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[Manually creating IAM for AWS] for steps to set the Cloud Credential Operator (CCO) to manual mode prior to installation. Use this mode in environments where the cloud identity and access management (IAM) APIs are not reachable, or if you prefer not to store an administrator-level credential secret in the cluster `kube-system` project.
+* See xref:../../authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc#manually-removing-cloud-creds_cco-mode-mint[Removing cloud provider credentials] to remove the administrator-level credential secret from the `kube-system` namespace in the cluster. The administrator-level credential is not required unless something that requires an administrator-level credential needs to be changed, for instance during an upgrade.
```

Comment:
If you changed this text in the 4.5 version, please update this text, too.

Reply:
@sagidlow please keep text consistent with version 4.5 and 4.6